src/main/java/io/neonbee/NeonBeeOptions.java

```diff
@@ -329,14 +322,14 @@ public Mutable setClustered(boolean clustered) {
             return this;
         }
 
-        public Mutable setServerVerticlePort(Integer serverVerticlePort) {
-            this.serverVerticlePort = serverVerticlePort;
+        public Mutable setServerPort(Integer serverPort) {
+            this.serverPort = serverPort;
             return this;
         }
 
         @Override
-        public Integer getServerVerticlePort() {
-            return this.serverVerticlePort;
+        public Integer getServerPort() {
+            return this.serverPort;
         }
 
         @Override
```

Comment:
unrelated: those methods are not tested: "setActiveProfiles, setClusterConfig (is never used), getClusterConfig (null case not tested)